src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/Performance/CSharpPreferLengthCountIsEmptyOverAny.Fixer.cs

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+using System.Composition;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.NetCore.Analyzers.Performance;
+using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
+
+namespace Microsoft.NetCore.CSharp.Analyzers.Performance
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp), Shared]
+    public sealed class CSharpPreferLengthCountIsEmptyOverAnyFixer : PreferLengthCountIsEmptyOverAnyFixer
+    {
+        protected override SyntaxNode? ReplaceAnyWithIsEmpty(SyntaxNode root, SyntaxNode node)
```

Comment:
Does this handle the niche case where `Any` is invoked as an ordinary method with `System.Linq.Enumerable.Any(collection)`?

Reply:
I don't know, but I recall working on an analyzer like this a couple of years ago and the intention was to not catch and arbitrary `Any` extension method.

Reply:
Yeah, I can see this becoming problematic. Maybe it is better just to handle LINQ's `Any` method?

Reply:
I added tests for invocation as a normal static method instead of as an extension method.